.github/workflows/lint.yml

```
@@ -37,11 +37,12 @@ jobs:
       - name: Build
         run: node --run build
 
-      - name: Lint
-        run: node --run lint
+      - parallel:
```

actionlint (1.7.12)

[error] step must run script with "run" section or run action with "uses" section

(syntax-check)